monkestation/code/modules/clothing/jobs/brig_physician.dm

```diff
@@ -0,0 +1,99 @@
+	desc = "A formerly retired security uniform pattern now brought back for those working the medical profession in the security apparatus."
+	name = "brig physician's jumpsuit"
+	icon = 'monkestation/icons/obj/clothing/jobs/brig_physician.dmi'
+	worn_icon = 'monkestation/icons/mob/clothing/jobs/brig_physician.dmi'
+	icon_state = "brigphys"
+	inhand_icon_state = "gy_suit"
+
+/obj/item/clothing/under/rank/security/brig_physician/skirt
+	name = "brig physician's jumpskirt"
+	desc = "A formerly retired security uniform pattern now brought back for those working the medical profession in the security apparatus."
+	icon_state = "brigphys_skirt"
+	icon = 'monkestation/icons/obj/clothing/jobs/brig_physician.dmi'
+	worn_icon = 'monkestation/icons/mob/clothing/jobs/brig_physician.dmi'
+	inhand_icon_state = "gy_suit"
+	body_parts_covered = CHEST|GROIN|ARMS
+	dying_key = DYE_REGISTRY_JUMPSKIRT
+	female_sprite_flags = FEMALE_UNIFORM_TOP_ONLY
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+
+/obj/item/clothing/under/rank/security/scrubs/sec
+	armor_type = /datum/armor/rank_medical
+	name = "brig physician's security scrubs"
+	desc = "It's made of a special fiber that provides minor protection against biohazards. This one is in security red."
+	icon = 'monkestation/icons/obj/clothing/jobs/brig_physician.dmi'
+	worn_icon = 'monkestation/icons/mob/clothing/jobs/brig_physician.dmi'
+	icon_state = "scrubssec"
+
+/obj/item/clothing/head/utility/surgerycap/sec
+	name = "security surgical cap"
+	desc = "A blue medical surgery cap to prevent the surgeon's hair from entering the insides of the patient!"
+	icon = 'monkestation/icons/obj/clothing/jobs/brig_physician.dmi'
+	worn_icon = 'monkestation/icons/mob/clothing/jobs/brig_physician.dmi'
+	icon_state = "surgicalcapsec"
+
+/obj/item/clothing/under/plasmaman/brig_physician
+	desc = "A new pattern plasmaman suit for those qualified as a brig physician."
+	name = "brig physician's envirosuit"
+	icon = 'monkestation/icons/obj/clothing/jobs/brig_physician.dmi'
+	worn_icon = 'monkestation/icons/mob/clothing/jobs/brig_physician.dmi'
+	icon_state = "brigphys_envirosuit"
+
+/obj/item/clothing/head/helmet/space/plasmaman/brig_physician
+	desc = "A new pattern plasmaman helmet for those qualified as a brig physician. This is still EVA rated too!"
+	name = "brig physician's envirosuit helmet"
+	icon = 'monkestation/icons/obj/clothing/jobs/brig_physician.dmi'
+	worn_icon = 'monkestation/icons/mob/clothing/jobs/brig_physician.dmi'
+	icon_state = "brigphys_envirohelm"
+
+/obj/item/clothing/gloves/color/plasmaman/brig_physician
+	desc = "A new pattern plasmaman glove set to match that drippy security grey suit."
+	name = "brig physician's envirogloves"
+	icon = 'monkestation/icons/obj/clothing/jobs/brig_physician.dmi'
+	worn_icon = 'monkestation/icons/mob/clothing/jobs/brig_physician.dmi'
+	icon_state = "brigphysplasma"
+	greyscale_colors = "#918F8C"
+
+/obj/item/clothing/suit/toggle/labcoat/brig_physician
```

Comment:
Are we not gonna add a bit of armor to the clothing items?

Reply:
Thinking about how I would want to add it. Not sure if they would go into the regular SecDrobe or if we need to build a whole new vendor quite yet.